base_layer/core/src/proto/transaction.proto

```diff
@@ -99,8 +99,17 @@ message OutputFeatures {
     uint64 maturity = 3;
     // Additional arbitrary info in coinbase transactions supplied by miners
     bytes coinbase_extra = 4;
+    // Features that are specific to a side chain
     SideChainFeature sidechain_feature = 5;
-    // Additional
+    // Encrypted Pedersen commitment openings (value and mask) for the output
+    EncryptedOpenings encrypted_openings = 6;
+    // The type of range proof used in the output
+    uint32 range_proof_type = 7;
```

Comment:
Would it make more sense to make this a proper `enum` for clarity and consistency?

Reply:
I would keep it like this as it is a simple conversion and consistent with `uint32 output_type = 2;`